### ISSUE TITLE
Error when window size < table size

### ISSUE THREAD

**Issue by Ben-Baert** [codito/habito#1]
If the window size is smaller than the table size, I get the following error:
```python
Traceback (most recent call last):
  File "/usr/local/bin/habito", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/habito/habito.py", line 111, in list
    r[0] = '\n'.join(wrap(r[0], max_col_width))
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/textwrap.py", line 380, in wrap
    return w.wrap(text)
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/textwrap.py", line 355, in wrap
    return self._wrap_chunks(chunks)
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/textwrap.py", line 249, in _wrap_chunks
    raise ValueError("invalid width %r (must be > 0)" % self.width)
ValueError: invalid width -27 (must be > 0)
```

It would be nice to have a cleaner error message or a dynamic table (e.g. pick nr of days depending on width).

**Comment by Ben-Baert** [comment]
It looks like this was dealt with in the last commit, but that the commit wasn't pushed to Pypi yet. Is this correct?

**Comment by codito** [comment]
Pushed [1.0a2](https://pypi.python.org/pypi/habito) to pypi. Let me know if it fixes this issue. Thanks for the report.

**Comment by Ben-Baert** [comment]
Hi, thanks for the quick reply! I just filed a pull request #2 that deals with the issue by adapting the table width to the terminal width. Let me know whether it meets your expectations and if not, what needs to be improved.

**Comment by codito** [comment]
Fixed with 295073c8390178d8b80b65b6cf8e30d9045223eb
